Lib/test/test_minidom.py

```diff
@@ -77,6 +77,51 @@ def testParseFromTextFile(self):
             dom.unlink()
             self.confirm(isinstance(dom, Document))
 
+    def testAttrModeSetsParamsAsAttrs(self):
+        attr = Attr("qName", "namespaceURI", "localName", "prefix")
+        self.assertEqual(
+            attr._name,
+            "qName",
+            "testAttrModeSetsParamsAsAttrs -- qName parameter does not match",
```

Comment:
Please remove all these textual descriptions, they don't add value over the stack trace.